{"name": "test_verify_phone_no[device1-phone_number0]", "status": "passed", "attachments": [{"name": "log", "source": "afcb2340-9149-42e3-8cd3-234127da5531-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "appium_driver", "value": "'device1'"}, {"name": "phone_number", "value": "['9021921234']"}], "start": 1638784697805, "stop": 1638784706335, "uuid": "c273d60d-a97d-44c3-9225-d5d09cb150c1", "historyId": "263255fd6f2ba9ad306038ea95fbb885", "testCaseId": "8f179043caacb0abcb24fe30a00081a0", "fullName": "test_cases.test_verify_ph_no.TestVerifyPhoneNo#test_verify_phone_no", "labels": [{"name": "tag", "value": "flaky(reruns=3)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('log_on_failure', 'appium_driver')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_verify_ph_no"}, {"name": "subSuite", "value": "TestVerifyPhoneNo"}, {"name": "host", "value": "EPINHYDW042A"}, {"name": "thread", "value": "4156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_verify_ph_no"}]}